tests/unit/test_connection_cmds.py

```diff
@@ -701,7 +701,8 @@ def GET_TEST_PATH_STR(filename):  # pylint: disable=invalid-name
     ['Verify connection command export no current conection. SEQ 1.7',
      {'general': [],
       'args': ['export']},
-     {'stderr': ['No server currently defined as current'],
+     {'stderr': ['No server specified for command "export" that requires '
+                 'a WBEM server'],
```

Comment:
Is this change done in the code?

Reply:
Yes in _context_obj.py pywbem_server property.  Modified slightly since this should be the current server rather than specified.

Reply:
DONE